graylog2-server/src/test/java/org/graylog2/alerts/AbstractAlertConditionTest.java

```diff
@@ -16,16 +16,17 @@
  */
 package org.graylog2.alerts;
 
+import com.google.common.collect.ImmutableMap;
 import org.graylog2.plugin.Tools;
 import org.graylog2.plugin.alarms.AlertCondition;
 import org.junit.Before;
 import org.junit.Ignore;
 import org.junit.Test;
 
 import java.util.Map;
+import java.util.Optional;
 
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.*;
```

Comment:
Please don't use wildcard imports.

Configuration for IntelliJ IDEA: http://stackoverflow.com/a/3348855